caas/kubernetes/provider/application/application.go

```diff
@@ -57,7 +57,7 @@ const (
 	containerProbeTimeout      = 1
 	containerProbePeriod       = 5
 	containerProbeSuccess      = 1
-	containerProbeFailure      = 1
+	containerProbeFailure      = 3
```

Comment:
can we comment what these represent

Reply:
its the number of failures

Reply:
The meaning of these fields is described in the k8s docs: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes